src/Components/Samples/BlazorUnitedApp/Pages/Counter.razor

```diff
@@ -15,4 +16,9 @@
     {
         currentCount++;
     }
+
+    class OwnRenderModeAttribute : RenderModeAttribute
+    {
+        public override IComponentRenderMode Mode => RenderMode.Server;
+    }
```

Comment:
We can remove this manual declaration once the `@rendermode` directive is implemented.